Tasks/Common/Sanitizer/Tests/L0Protect-ScriptArguments.Throws.ps1

```diff
@@ -0,0 +1,21 @@
+[CmdletBinding()]
+param()
+
+Set-Item -Path env:AZP_75787_ENABLE_NEW_LOGIC -Value 'true'
+
+. $PSScriptRoot\..\..\..\..\Tests\lib\Initialize-Test.ps1
+. $PSScriptRoot\..\ArgumentsSanitizer.ps1
+
+$inputArgsSuites = @(
+    'test; whoami',
+    'test && whoami',
+    'echo "$(rm ./somedir)"'
```

Comment:
Are we know other cases we should cover here?

Reply:
I remember one case with vertical bar `|`, let me try to reproduce it

Reply:
Vertical bar is scenario for Linux, so I'll add it